app/scripts/migrations/119.test.ts

```diff
@@ -0,0 +1,128 @@
+import { createMockInternalAccount } from '../../../test/jest/mocks';
+import { migrate, version } from './119';
+
+const mockTimeStamp = 1716972230;
+jest.useFakeTimers().setSystemTime(new Date(mockTimeStamp));
+
+const oldVersion = 118;
+const newVersion = 119;
+
+describe('migration #119', () => {
+  it('updates the version metadata', async () => {
+    const oldStorage = {
+      meta: { version: oldVersion },
+      data: {},
+    };
+
+    const newStorage = await migrate(oldStorage);
+    expect(newStorage.meta).toStrictEqual({ version });
+  });
+
+  it('returns the default state if the AccountsController state is undefined', async () => {
+    const oldStorage = {
+      meta: {
+        version: oldVersion,
+      },
+      data: {
+        AccountsController: {
+          internalAccounts: {
+            accounts: {},
+            selectedAccount: '',
+          },
+        },
+      },
+    };
+
+    const newStorage = await migrate(oldStorage);
+    expect(newStorage).toStrictEqual({
+      meta: {
+        version: newVersion,
+      },
+      data: {
+        AccountsController: {
+          internalAccounts: {
+            accounts: {},
+            selectedAccount: '',
+          },
+        },
+      },
+    });
+    expect(newStorage);
+  });
```

Comment:
I feel like "undefined" should be "default state" instead in the title.

Also, nit pick, but we could have used a variable like:
```ts
const defaultState = {
  internalAccounts: {
    accounts: {},
    selectedAccount: '',
  }
};
```

and use it like `AccountsController: defaultState` in both the `oldStorage` and `toStrictEqual` to emphasis on the state being the same.

(feel free to ignore this one, I just thought it was great for readibility)

Reply:
or use the same `mockAccountsControllerState` name (same as some other tests in this file)